{
  "uid" : "ff409dccdd6896bf",
  "name" : "token match",
  "fullName" : "Contract: ERC20FlashMint maxFlashLoan.token match",
  "historyId" : "9ded4fc02ea204e41878dd8e83157578",
  "time" : {
    "start" : 1635554997544,
    "stop" : 1635554998521,
    "duration" : 977
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7bfa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7bfa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC20/extensions/ERC20FlashMint.test.js:20:46)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ERC20FlashMint maxFlashLoan"
  }, {
    "name" : "testClass",
    "value" : "Contract: ERC20FlashMint maxFlashLoan"
  }, {
    "name" : "testMethod",
    "value" : "token match"
  }, {
    "name" : "package",
    "value" : "Contract: ERC20FlashMint maxFlashLoan"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "c9fcaee738ce8570",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a2adf490>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554930465,
        "stop" : 1635554931448,
        "duration" : 983
      }
    }, {
      "uid" : "96b4fd47650802c4",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0437610>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554915528,
        "stop" : 1635554916124,
        "duration" : 596
      }
    }, {
      "uid" : "5c6519beb55f8283",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0670fa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554904431,
        "stop" : 1635554905044,
        "duration" : 613
      }
    }, {
      "uid" : "6c27008476059edb",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0682be0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554899928,
        "stop" : 1635554900514,
        "duration" : 586
      }
    }, {
      "uid" : "d820e0271caac06f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07ca400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554860127,
        "stop" : 1635554861090,
        "duration" : 963
      }
    }, {
      "uid" : "8b18a80c0f167990",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0689970>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554835579,
        "stop" : 1635554836168,
        "duration" : 589
      }
    }, {
      "uid" : "8ddf824945c40c42",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a042b6d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554822682,
        "stop" : 1635554823280,
        "duration" : 598
      }
    }, {
      "uid" : "459ad28cffaf8981",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0427a90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554819787,
        "stop" : 1635554820375,
        "duration" : 588
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "3370a6f53fa956fa",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/3370a6f53fa956fa",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551255007,
          "stop" : 1635551255080,
          "duration" : 73
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "ff409dccdd6896bf.json",
  "parameterValues" : [ ]
}